### PR TITLE
MGDOBR-804 : Add script to delete managed connectors

### PR DESCRIPTION
**Pull request by Srihari1192**
[MGDOBR-804](https://issues.redhat.com/browse/MGDOBR-804)

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] The layers in the `kustomize` folder have been updated accordingly.
- [ ] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

**Comment by sutaakar** [comment]
@Srihari1192 Please add execute permissions for the script

File: dev/bin/interaction/delete-managed-connectors.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Script uses the MANAGED_CONNECTORS_CONTROL_PLANE_URL env var from localconfig as a Connectors base path
+# This requires the ocm command: https://github.com/openshift-online/ocm-cli
+# You will need to login with `ocm login --token [...]`.
+
+. $(dirname "${BASH_SOURCE[0]}")/../configure.sh
+
+export COS_BASE_PATH=${MANAGED_CONNECTORS_CONTROL_PLANE_URL}/api/connector_mgmt/v1
+export CONNECTORS_BASE=${COS_BASE_PATH}/kafka_connectors
+
+MC_IDS=$(curl -L --insecure --oauth2-bearer "$(ocm token)" "${CONNECTORS_BASE}" | jq -r '(now - 12*60*60 | strflocaltime("%Y-%m-%dT%H:%M:%S")) as $older  | .items[] | select(.name | startswith("ci-"))| select((.created_at | sub("-(?<d>[0-9]-)";"-0\(.d)")) <= $older)| .id')
```
Comment (sutaakar) [inline dev/bin/interaction/delete-managed-connectors.sh:14]: Please filter also just connectors with owner `openbridge_kafka_supporting`

**Comment by sutaakar** [comment]
Few minor comment, otherwise looks good

File: dev/bin/interaction/delete-managed-connectors.sh (hunk 1 above)
Comment (Srihari1192) [reply dev/bin/interaction/delete-managed-connectors.sh]: Done

**Review by sutaakar — APPROVED** [review]
Looks good to me, thanks

**Review by jiripetrlik — APPROVED** [review]

**Pull request merged**